cards/wallet/cardstack/static-model.js

```diff
@@ -45,7 +45,7 @@ factory.addResource('content-types', 'wallets')
   ]);
 
   factory.addResource('grants', 'wallet-self-grant')
-    .withRelated('who', [{ type: 'fields', id: 'user' }])
+    .withRelated('who', [{ type: 'groups', id: 'everyone' }])
```

Comment:
just to be clear, this lets anyone view anyone elses' wallet...

Reply:
Indeed. But there might not actually be a `user`, or any way to uniquely identify someone other than the fact that they've enabled Metamask. I'm curious to know if there's a way to lock wallets down via grants without knowing the user's identity

Reply:
yeah, i guess I'd prefer to see wallets go away than to open up everyone's wallets to the world. Especially with this being an open source project.

if we dont know the user's identity, there really isn't a way to lock things down. metamask is really not a substitute for identity, as someone could monkey patch the metamask provider in a chrome debugger to masquerade as a user to the hub.

Reply:
or i guess it not only allows anyone to read anyone else wallet, but any one to be able to write fields into anyone else' wallet...